DBScripts/00_CreateDBAndTables.sql

```diff
@@ -56,7 +56,7 @@ CREATE TABLE IF NOT EXISTS Dorms(
 );
 
 CREATE TABLE IF NOT EXISTS Rooms(
-	room_number VARCHAR(4) NOT NULL,
+	room_number VARCHAR(8) NOT NULL,
```

Comment:
This needs to be changed in the GroupWishlist and StudentWishlist tables as well

Reply:
Done